### ISSUE TITLE
Add types

### ISSUE THREAD

**Issue by zachazar** [greguz/fastify-no-additional-properties#13]
This plugin is great! Thank you for creating this 🙌.

I'm using it in a TypeScript project and I added the following types locally. Would you want to add these to [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)? Unfortunately I don't have the bandwidth to do it now but could maybe create a PR in a few months.

Related issue: https://github.com/greguz/fastify-no-additional-properties/issues/12

```ts
import { FastifyPluginCallback } from 'fastify'

declare module 'fastify-no-additional-properties' {
  interface FastifyNoAdditionalProperties {
    /**
     * If true, update the request body schema.
     * @default true
     */
    body?: boolean
    /**
     * If true, update the request headers schema.
     * @default false
     */
    headers?: boolean
    /**
     * If true, update the URL parameters schema.
     * @default false
     */
    params?: boolean
    /**
     * If true, update the URL querystring schema.
     * @default true
     */
    query?: boolean
    /**
     * If true, update all response schemas.
     * @default false
     */
    response?: boolean
  }

  export const fastifyNoAdditionalProperties: FastifyPluginCallback<FastifyNoAdditionalProperties>

  export default fastifyNoAdditionalProperties
}
```

**Comment by greguz** [comment]
Sorry for the late response, my GitHub missed some notifications... Next days I'll publish a new version with a fix (other PR) and those types.

**Comment by greguz** [comment]
Closed with version `v2.4.0`.

**Comment by zachazar** [comment]
Thanks @greguz 🙌 !
